CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#187 <https://github.com/openclimatedata/pymagicc/pull/187>`_) Added `pymagicc.io.join_timeseries`
```

Comment:
Maybe add something like "simplifies joining/merging scenarios to create custom scenarios"?